_episodes/07-resources.md

```diff
@@ -117,20 +117,24 @@ Rules claiming more threads will be scaled down.
 
 Our pipeline ran in parallel and finished roughly 4 times as quickly! The
 takeaway here is that all we need to do to scale from a serial pipeline is
-run `snakemake` with the `-j` option. By analysing the dependencies between
+run `snakemake` with the `-c 4` option. By analysing the dependencies between
 rules, Snakemake automatically identifies which tasks can run at the same
 time. All you need to do is describe your workflow and Snakemake does the
 rest.
 
-Note you can also use `snakemake --cores 4` or `snakemake --jobs 4`. The
-`-j`, `--cores` and `--jobs` arguments all mean the same thing.
+> ## Note
+> You can also use `snakemake --cores all` or `snakemake --jobs all`. The
+> `-c`, `-j`, `--cores` and `--jobs` arguments all mean the same thing
+> when running locally on your own machine.
+{: .callout}
 
 > ## Self-documention
 >
 > Using the long-form of command-line arguments can be useful in scripts. They
-> make the code more understandable since you don't need to remember
-> what `-j` does. `--cores` is most useful in scripts that run snakemake
-> locally, while `--jobs` is most often used when running on a HPC cluster.
+> make the code more understandable since you don't need to remember what
+> `-c` does. If you forget use `snakemake -h` to get the help
+> commands for what each of these do, and how they differ when running on a
+> remote computational resource.
```

Comment:
I worry about putting it this way, since `-j` does mean something else on compute clusters from the documentation?

<img width="917" alt="image" src="https://user-images.githubusercontent.com/4336207/194419277-eb354a4d-0557-471e-bb48-7b6fb8a84a2d.png">

Reply:
@reid-a suggested (+1) that we can drop all mentions of `-j` and any discussion pertaining to it until the compute cluster configuration is covered.

Reply:
That being the case, we can (and IMO should) introduce the "-j" argument when we get to the cluster case, and use "-c" before then.

Reply:
Created a new issue for this.  Probably best to address it after resolving #29